### ISSUE TITLE
No Disk I/O charts

### ISSUE THREAD

**Issue by Pootch** [henrygd/beszel#183]
Can somebody help with the disk i/o chart. It does not show any stats.

My docker compose:

```
services:
  beszel:
    container_name: Beszel
    image: henrygd/beszel:latest
    volumes:
      - /volume1/docker/beszel:/beszel_data:rw
    ports:
      - 8095:8090
    restart: on-failure:5

  beszel-agent:
    image: henrygd/beszel-agent
    container_name: Beszel-Agent
    network_mode: host
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
    environment:
      PORT: 45876
      KEY: mykey
      FILESYSTEM: /dev/vg1/volume_1
    restart: on-failure:5
```

**Comment by henrygd** [comment]
Please post your agent logs and the output of these commands:

```
lsblk
```

```
cat /proc/diskstats
```

**Comment by Pootch** [comment]
[_Beszel-Agent_logs.txt](https://github.com/user-attachments/files/17135333/_Beszel-Agent_logs.txt)
[filesystem.txt](https://github.com/user-attachments/files/17135334/filesystem.txt)

Added the log & output.
lsblk is not available on my system so I've used df -h

**Comment by henrygd** [comment]
Looks like it's probably mapped to `dm-1`, so try `FILESYSTEM=dm-1`. Or just remove the `FILESYSTEM` env var and see what happens.

If that doesn't work, use a bad value like `FILESYSTEM=/xyzzy`, which will print all available partitions in the agent logs. Then copy/paste that here please.

**Comment by Pootch** [comment]
"/dev/vg1/volume_1" as Filesystem seems to be right, but still no Disk I/O values.

Logs with Filesystem=dm-1
[Beszel-Agent-2024-09-26T05-44-52.log](https://github.com/user-attachments/files/17143346/Beszel-Agent-2024-09-26T05-44-52.log)

**Comment by henrygd** [comment]
I think I figured out the issue and fixed it in 0.5.0.

No rush, but please update when you can and let me know if the I/O values start coming in.

**Comment by Pootch** [comment]
Hello. 
With the new build I get the Disk I/O charts with "FILESYSTEM: /dev/vg1/volume_1"

Thanks for help and the fast fix.
